src/lib/tabcomplete.ts

```diff
@@ -170,7 +170,7 @@ function onKeyPress(e, options) {
                      /* Space should not be added when there is only 1 match
                             or if there is already a space following the caret position */
                      let space = (match.matches.length > 1 || last.length && last.substr(0, 1) === " ") ? "" : (first.trim().length === 0 ? ": " : " ");
-                     $this.val(first + match.value + space + last);
+                     $this.val(first + '@"' + match.value + '/' + player_cache.lookup_by_username(match.value).id + '"' + space + last);
```

Comment:
I don't mind making @ completion a thing, but would it be possible to support the existing name completion *and* @ prefixed tab completion? Also, when I type '@a<tab>` I get @@"anoek/1" is that expected, or should it just be with one @? (I don't really care in either case)

Reply:
> but would it be possible to support the existing name completion _and_ @ prefixed tab completion?

An option would be to do the @"" thing only when the username is already completed (e.g. `a =tab> anoek =tab> @"anoek/1"`), but I wasn't able to check if a completion is needed. I don't understand the tab-completion code well enough to find out how to do it.

> when I type `@a<tab>` I get `@@"anoek/1"` is that expected

The `@@"anoek/1"` is the linked equivalent to `@anoek`, so I think leaving the `@ is the expected behavior. I thought  about what to do when `@"a =tab>`, but decided to go the easy way by not checking for it.